app/src/auth/components/RegisterPage/index.jsx

```diff
@@ -125,68 +129,89 @@ const RegisterPage = ({ location: { search, pathname }, history: { replace } }:
         ))
     }, [form, setSessionToken, mountedRef])
 
+    const strength = usePasswordStrength(form.password)
```

Comment:
This is new. `usePasswordStrength` hook gives you a number. It's a hook. We know the drill.